src/modules/test/RecipeInvoker.h

```diff
@@ -4,7 +4,7 @@
 #ifndef RECIPEINVOKER_H
 #define RECIPEINVOKER_H
 
-static const std::string g_defaultClient = "RecipeInvoker";
+static const std::string g_defaultClient = "Azure OsConfig";
```

Comment:
This will still fail the validation used by the module here: https://github.com/Azure/azure-osconfig/blob/main/src/common/commonutils/CommonUtils.cpp#L12

We will need a full version string in this form: `Azure OSConfig <model version>;<major>.<minor>.<patch>.<yyyymmdd><build>`

Reply:
That form will be the same string as is passed from the platform to the modules during `MmiOpen()` at runtime.

Reply:
Good catch!